cw/worker.py

```diff
@@ -108,6 +109,7 @@ def landsat_dataset(self, spectrum, x, y, t, ubid, specs_url, tiles_url):
         ds.coords['t']        = (('t'), pd.to_datetime([t['acquired'] for t in tiles]))
         ds.coords['source']   = (('t'), [t['source'] for t in tiles])
         ds.coords['acquired'] = (('t'), [t['acquired'] for t in tiles])
+        # the following can probably go away, since it gets wiped out during xarray combine_first()
```

Comment:
It seems strange that this would get wiped out but source and acquired would not...